src/utils/props.js

```diff
@@ -1,9 +1,9 @@
-
 export const genericSyntaxes = {
   a11yTitle: 'a user friendly label for screen readers',
   alignSelf: ['start', 'center', 'end', 'stretch'],
   gridArea: 'a parent grid area name',
   margin: [
+    'none',
```

Comment:
'none' only applies to Heading and Paragraph. The other places we accept margin don't use this value.

Reply:
Good catch! I'll take another look